src/map/utils/mobutils.cpp

```diff
@@ -705,6 +706,33 @@ void SetupJob(CMobEntity* PMob)
             PMob->defaultMobMod(MOBMOD_BUFF_CHANCE, 100);
     }
 
+    // Just a fallback at the moment
+    switch(sJob)
+    {
+        case JOB_BLM:
+        case JOB_WHM:
+        case JOB_RDM:
+        case JOB_DRK:
+        case JOB_BRD:
+        case JOB_BLU:
+        case JOB_SMN:
+        case JOB_BRD:
```

Comment:
you listed JOB_BRD twice (gj travis https://travis-ci.org/DarkstarProject/darkstar/builds/146070689#L1986)